items.c

```diff
@@ -999,19 +999,8 @@ item *do_item_get(const char *key, const size_t nkey, const uint32_t hv, LIBEVEN
             it = NULL;
         }*/
     }
-    int was_found = 0;
 
-    if (settings.verbose > 2) {
-        int ii;
-        if (it == NULL) {
-            fprintf(stderr, "> NOT FOUND ");
-        } else if (was_found) {
```

Comment:
I think the actual fix is to change this to a bare `} else {` and not change anything else.

It looks like I'd made the same mistake, but left the `else if (was_found` after moving the code back.